tutorials/mongoose/fix-buffering-timed-out.md

```diff
@@ -0,0 +1,45 @@
+This error at its core is a result of not being able to connect to MongoDB.
```

Comment:
"This error happens because you're trying to use a model whose connection hasn't connected to MongoDB."